scripts/known_good.json

```
@@ -38,7 +38,7 @@
                 "-DSPIRV-Headers_SOURCE_DIR={repo_dir}/../SPIRV-Headers",
                 "-DSPIRV_WERROR=OFF"
```

Skipping tests appears to fix the build issue.

```suggestion
                "-DSPIRV_WERROR=OFF",
                "-DSPIRV_SKIP_TESTS=ON"
```

---

Actually please skip building the tests and skip building the executables. We only need the libraries.

```
        {
            "name": "SPIRV-Tools",
            "url": "https://github.com/KhronosGroup/SPIRV-Tools.git",
            "sub_dir": "SPIRV-Tools",
            "build_dir": "SPIRV-Tools/build",
            "install_dir": "SPIRV-Tools/build/install",
            "cmake_options": [
                "-DSPIRV-Headers_SOURCE_DIR={repo_dir}/../SPIRV-Headers",
                "-DSPIRV_WERROR=OFF",
                "-DSPIRV_SKIP_TESTS=ON",
                "-DSPIRV_SKIP_EXECUTABLES=ON"
            ],
            "commit": "d33bea584707c3509065010c45d9b79c54c8376b"
        },
```

---

aww, beat me to finding this... yes, do this :100: